templates/Slack/SlackMessage.html

```diff
@@ -0,0 +1,59 @@
+<style>
```

Comment:
Wouldn't those styles affect other unrelated result templates ?

Reply:
I add a class to the main `div` of the template to scope the css rules better !